gw_eccentricity/eccDefinition.py

```diff
@@ -237,6 +237,16 @@ def __init__(self, dataDict, num_orbits_to_exclude_before_merger=2,
                 methods. See
                 eccDefinitionUsingFrequencyFits.get_default_kwargs_for_fits_methods
                 for allowed keys.
+
+            set_failures_to_zero : bool, default=False
+                The code normally raises an exception if sufficient number of
+                extrema are not found. This can happen for various reasons
+                including when the eccentricity is too small for some methods
+                (like the Amplitude method) to measure. See e.g. Fig.4 of
+                arxiv.2302.11257. If `set_failures_to_zero` is set to True, we
+                assume that small eccentricity is the cause, and set the
```

Comment:
Actually we should be more clear:
assume that small eccentricity is the cause as long as the waveform has at least 5 orbits
Question: What is the actual rule. We take a waveform, truncate 2 orbits at the end, and then look for 5 orbits in this test? So, is it min 7 orbits? Whatever it is, give the total number here (7?), and in the code where you have stuff like 5 orbits, explain what that means via a comment.

Reply:
Done!